test/rbenv.bats

```diff
@@ -149,39 +149,63 @@ OUT
 
 @test "not enough arguments for rbenv-install" {
   stub_ruby_build
+  stub rbenv-help 'install : echo "Usage: rbenv install"'
```

Comment:
Thanks, but with this stub you're just verifying that `rbenv-help` gets called. Because fake rbenv-help stubs its own output, we're never testing that this help text is indeed present within ruby-build.

Instead, try a different stubbing approach. Inside the `setup` block, create a new executable called `${TMP}/bin/rbenv-help` with the contents:

```sh
#!/bin/bash
set -e
cmd="${1?}"
file="$(which "rbenv-$cmd")"
sed -ne '/^#/!q;s/.//;s/.//;1,4d;p' < "$file"
```

This simulates real `rbenv-help` by extracting the help text off the top of a `rbenv-<command>` exectuable file.